Sources/Jobs/EncodableJob.swift

```diff
@@ -19,9 +19,9 @@ struct EncodableJob<Parameters: Codable & Sendable>: Encodable, Sendable {
     let id: JobIdentifier<Parameters>
     let data: JobInstanceData<Parameters>
 
-    init(id: JobIdentifier<Parameters>, parameters: Parameters, queuedAt: Date) {
+    init(id: JobIdentifier<Parameters>, parameters: Parameters, queuedAt: Date, attempts: Int) {
```

Comment:
should attempts get a default value to prevent jobs without this parameter from failing to decode?

Reply:
I suppose we do need to pass `attempts` since data is of type `JobInstanceData` which defines `attempts`?